vizro-core/src/vizro/static/css/layout.css

```diff
@@ -138,25 +138,16 @@ div.dashboard_container .custom-tooltip {
   padding: 4px;
 }
 
-.nav-icon {
-  filter: var(--fill-accordion-button);
-  width: 24px;
-  height: 24px;
-}
-
-.nav-icon-text {
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  gap: 6px;
-  justify-content: center;
+.icon-button.btn.btn-primary.active .material-symbols-outlined {
+  color: var(--text-active);
 }
-
-.icon-button.btn.btn-primary.active {
-  background-color: var(--state-overlays-selected);
-}
-
 .loading-container {
   height: 100%;
   width: 100%;
 }
+
+.mantine-Tooltip-tooltip {
```

Comment:
I think if you give the relevant mantine component a className, you can reference it here, instead of relying on the default mantine className. I would prefer that :)